bin/disco_snapshot.py

```diff
@@ -98,15 +99,19 @@ def run():
     elif args.mode == "cleanup":
         aws.disco_storage.cleanup_ebs_snapshots(args.keep)
     elif args.mode == "capture":
-        instances = instances_from_args(aws, args)
-        if not instances:
-            logging.warning("No instances found")
-        for instance in instances:
-            return_code, output = aws.remotecmd(
-                instance, ["sudo /opt/wgen/bin/take_snapshot.sh"], user="snapshot")
-            if return_code:
-                raise Exception("Failed to snapshot instance {0}:\n {1}\n".format(instance, output))
-            logging.info("Successfully snapshotted %s", instance)
+        if args.volume_id:
+            snapshot_id = aws.disco_storage.take_snapshot(args.volume_id)
+            logging.info("Successfully created snapshot: %s", snapshot_id)
```

Comment:
Turns out the `take_snapshot.sh` script doesn't actually need the `snapshot_id`. So we'll just log it here instead of printing it out.